src/Defines.h

```diff
@@ -594,9 +594,9 @@ inline bool IsBlockMaterial_PLANTS(BLOCKTYPE a_BlockType)
 		case E_BLOCK_CARROTS:
 		case E_BLOCK_POTATOES:
 		// Blocks which don't exist in Cuberite yet.
-//  case 199:
-//  case 200:
-//  case 207:
+		case 199:
+		case 200:
+		case 207:
```

Comment:
There aren't because these blocks don't exist in cuberite yet. Should I add them in BlockID.h (I think there) or should I add comments what they are?